### PR TITLE
add github templates to repository

### PR DESCRIPTION
**Pull request by prempador**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Bug Report
+
+(A clear and concise description of what the bug is)
+
+## To Reproduce
+
+(Write your steps here:)
+
+1. Step 1...
+1. Step 2...
```
Comment (pgaubatz) [inline .github/ISSUE_TEMPLATE/bug.md:16]: 2.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+1. Step 3...
```
Comment (pgaubatz) [inline .github/ISSUE_TEMPLATE/bug.md:17]: 3.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+name: Bug Report
+about: Submit a bug report to help us improve
+labels: 'bug, needs triage'
+---
+
+## Bug Report
+
+(A clear and concise description of what the bug is)
+
+## To Reproduce
```
Comment (pgaubatz) [inline .github/ISSUE_TEMPLATE/bug.md:11]: steps to reproduce

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (prempador) [reply .github/ISSUE_TEMPLATE/bug.md:16]: Works both ways to my surprise. But I changed it.

**Pull request merged**